controllers/feeds/getFeedsV2.js

```diff
@@ -182,7 +192,7 @@ module.exports = async (req, res) => {
             item.is_self =
               item.actor.id === req.userId || item.actor.id === myAnonymousUser?.user_id;
             let newItem = await modifyFeedIsFollowingTarget(item, req.userId);
-            newItem = await modifyAnonimityPost(newItem);
+            newItem = await modifyAnonimityPost(newItem, isBlurredPost);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyAnonimityPost` function call is updated with an additional argument `isBlurredPost`. This change is necessary for the new feature of blurring posts for certain users. The changes look good.
```diff
- newItem = await modifyAnonimityPost(newItem);
+ newItem = await modifyAnonimityPost(newItem, isBlurredPost);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->